### PR TITLE
[Colors & Styles] MS Typography

### PR DESCRIPTION
**Pull request by madeleinesong**
text components that mirror the figma and provide specified text formats.

### CODE REVIEW

File: src/Typography/Body.react.tsx [hunk 1]
```diff
@@ -39,7 +39,18 @@ const Body: React.FC<BodyProps> = ({
     }
   }
 
-  const fontFamily = bold ? "Poppins_600SemiBold" : "Poppins_400Regular";
+  let fontFamily = bold ? "Inter_400Regular" : "Poppins_400Regular";
```
Comment (toriqiu) [inline src/Typography/Body.react.tsx:42]: Should these be semibold/regular versions of Inter_400?

File: src/Typography/Body.react.tsx [hunk 2]
```diff
@@ -39,7 +39,18 @@ const Body: React.FC<BodyProps> = ({
...
+  switch (size) {
+    case 1:
+      fontFamily = "Inter_400Regular";
+      break;
+    case 2:
+      fontFamily = "Inter_600SemiBold";
+      break;
+    case 3:
+      fontFamily = "Poppins_400Regular";
```
Comment (toriqiu) [inline src/Typography/Body.react.tsx:51]: Is this supposed to be Inter instead of Poppins?

File: src/Typography/Header.react.tsx [hunk 3]
```diff
@@ -43,16 +43,20 @@ const Header: React.FC<HeaderProps> = ({
...
-  let fontFamily = "Poppins_700Bold";
+  let fontFamily = "Inter_700Bold";
   switch (size) {
     case 1:
-      fontFamily = "Poppins_700Bold";
+      fontFamily = "Inter_700Bold";
       break;
     case 2:
+      fontFamily = "Inter_600SemiBold";
+      break;
```
Comment (toriqiu) [inline src/Typography/Header.react.tsx:53]: Should we just use 2 cases here for Inter_700Bold and Inter_600SemiBold?

**Review by toriqiu — APPROVED** [review]

File: src/Typography/Body.react.tsx (hunk 2 above)
Comment (madeleinesong) [reply src/Typography/Body.react.tsx:51]: no

File: src/Typography/Body.react.tsx (hunk 1 above)
Comment (madeleinesong) [reply src/Typography/Body.react.tsx:42]: i don't think so

**Pull request merged**